lib/src/main/java/growthbook/sdk/java/Constants.java

```diff
@@ -0,0 +1,15 @@
+package growthbook.sdk.java;
+
+interface Constants {
+    String GET_METHOD = "GET";
+    String X_SSE_SUPPORT = "x-sse-support";
+    String ACCEPT = "Accept";
+    String SSE_HEADER = "text/event-stream";
+    String ENABLED = "enabled";
+    String ENCRYPTED_FEATURES_KEY = "encryptedFeatures";
+    String ENCRYPTED_SAVED_GROUPS_KEY = "encryptedSavedGroups";
+    String FEATURE_KEY = "features";
+    String SAVED_GROUP_KEY = "savedGroups";
+    String DATA_SSE_KEY = "data:";
+    Integer QUANTITY_TO_CUT_SSE = 5;
+}
```

Comment:
please move these constant definitions to the respective files and not create a dedicated interface for constants. Makes it easy to maintain.